### PR TITLE
templates > .github and create release_template

### PR DESCRIPTION
**Pull request by michaelstingl**
I would like to have the release steps under version control to constantly improve the process. And we could put all templates to the .github folder same as it's in the core repo.

@davivel @davigonz @jesmrec What do you think?

**Open taks:**

- [x] make version generic
- [ ] Merge with "Squash and merge" option

### CODE REVIEW

File: .github/release_template.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+AC:
+
+ - [ ] [GIT] Create branch release_0.9.16 in owncloud/android-library from master  (almost feature-frozen)
+ - [ ] [GIT] Create branch release_2.3.0 in owncloud/android from master (almost feature-frozen)
+ - [ ] [DEV] Update version number and name in AndroidManifest.xml in android module
+ - [ ] [DEV] Update version number and name in AndroidManifest.xml in oc_jb_workaround module
+ - [ ] [DIS] Create changelog file (< 500 chars) and add to CHANGELOG.md in owncloud/android
+ - [ ] [QA] Design Test plan  @jesmrec
+ - [ ] [QA] Regression Test plan @jesmrec
```
Comment (jesmrec) [inline .github/release_template.md]: Thanks for your trust :), but i would not assign tasks in a template.

File: .github/release_template.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+ - [ ] [GIT] Merge branch release_0.9.16 in owncloud/android-library, into master
+ - [ ] [GIT] Merge branch release_2.3.0 in owncloud/android, into master
+ - [ ] [DIS] Generate test APKs and BAR files from branch master in owncloud/android
+ - [ ] [GIT] Create and sign tag oc-android-2.3.0 in HEAD commit of stable master branch, in owncloud/android
+ - [ ] [GIT] Create and sign tag oc-android-library-0.9.16 in HEAD commit of stable master branch, in owncloud/android-library
```
Comment (jesmrec) [inline .github/release_template.md]: should the release template indicate a version id? maybe the template should have a generic value, and when the release issue is created, set some of the tasks with the particulars of the release

Comment (michaelstingl) [inline .github/release_template.md]: In the client repo, everything is generic or an example:
https://github.com/owncloud/client/blob/organize-templates/.github/release_template.md

Comment (michaelstingl) [inline .github/release_template.md]: @jesmrec Could you replace the specific version with a generic string?

**Comment by jesmrec** [comment]
All ok for me.

**Comment by davivel** [comment]
Will merge when Travis finishes.

**Pull request merged**